boltdesignsystem/bolt
# DS 886: Teaser Updates

## Jira

https://pegadigitalit.atlassian.net/browse/DS-886

## Summary

The Teaser was updated with two props that can more completely replace the Card Component.

## Details

**1. A `spacing` prop with the following options was added**
- small
- medium
- large

**2. A `border_radius` prop with the following options was added**
- none
- small
- large

**3. A test page was created with all new props options `/pattern-lab/?p=tests-teaser-spacing-and-border-radius`**

## How to test

Pull the branch. Check the test page and confirm that the new props work as expected and the component can more completely replace old Card.

## Release notes

The Teaser component has two new props (`spacing`, `border_radius`) that can be applied to the card variant teaser .

### Visual changes

The spacing and the corner roundness can be adjusted in the card variant teaser component.
